lib/build.ts

```diff
@@ -121,20 +127,35 @@ export async function buildApp(src: string, dest: string, network: string = "mai
       new_logs.forEach(log.send);
 
       logs.push(...new_logs);
+
       // write to dest
       let new_file_name = path.relative(path.join(src, "widget"), file).replace("/", ".");
       new_file_name = new_file_name.substring(0, new_file_name.length - path.extname(file).length);
       new_file_name += ".jsx";
 
       const new_file_path = path.join(dest, "src", "widget", new_file_name);
+      new_build_files.push(new_file_path);
+
       await outputFile(new_file_path, new_file.code);
     }
+
     loadingWidgets.finish(`Transpiled ${widgets.length} widgets`);
   } catch (e) {
     loadingWidgets.error(`Failed to transpile widgets`);
     throw e;
   }
 
+  // remove unnecessary build files
+  await loopThroughFiles(path.join(dest, "src", "widget"), async (file: string) => {
```

Comment:
Can you write a test in[test/unit/build.ts](https://github.com/NEARBuilders/bos-workspace/blob/main/tests/unit/build.ts)

'should remove unnecessary files from build on rebuild`